.travis.yml

```diff
@@ -29,13 +29,12 @@ services:
 before_install:
   - git --version
   - mongod --version
-  - npm install -g adapt-cli
-  - adapt --version
   - npm install -g grunt-cli
 
 install:
   - npm config set spin false
-  - npm install
+  - npm install --production
```

Comment:
These needs to remain without the `--production` flag so that the tests can run.